src/cljam/io/util.clj

```diff
@@ -136,38 +140,3 @@
     #"(?i)\.bcf$" :bcf
     #"(?i)\.bed" :bed
     (throw (IllegalArgumentException. "Invalid file type"))))
-
-(defn ^Closeable reader
```

Comment:
How about leaving `reader/writer` functions here?
It's true that they're unnecessary for developing apps, but I think those versatile functions are still useful for REPL.